### ISSUE TITLE
User registration/password setting broken

### ISSUE THREAD

**Issue by schniggie** [natlas/natlas#668]
**Version:**
latest

**Describe the bug**
User registration, especially setting a password seems broken, got the same issue using the latest docker development setup on different docker environment test setups. Registration of a new admin user, by the provided docker exec instruction works. However, the login afterwards with the provided registration details fails. According to the users page within the WebUI it seems the user was successfully created. There is one change I had to add to the `docker-compose.yml `provided by you. I added `DB_AUTO_UPGRADE=True` to the environment of the natlas-server service, to get the service successfully up.

**To Reproduce**

1. `git clone https://github.com/natlas/natlas.git`
2. Add `DB_AUTO_UPGRADE=True` to `docker-compose.yml` within '`environment`' of `natlas-server` service
3. `docker-compose up -d`
4. `docker exec -e SERVER_NAME=example.com -it $(docker ps | grep natlas-server | cut -d' ' -f1) flask user new --admin`
5. Register a new admin with the token URL provided by step 4
6. Logout and try logging in with the user just created. Fails.

**Comment by 0xdade** [comment]
Hey there. I'm not actively working on this project very much at the moment but to help debug, could you capture the Network tab in dev tools when you try to login? What does the failure look like? Is it just a login error saying username/password invalid? Or a 500 error? Or redirecting to the wrong URL?

**Comment by schniggie** [comment]
POST to /auth/login is answered with

```
HTTP/1.0 302 FOUND
Content-Type: text/html; charset=utf-8
Content-Length: 229
Location: http://xxxremoved.tld:5000/auth/login
Set-Cookie: remember_token=; Expires=Thu, 01-Jan-1970 00:00:00 GMT; Max-Age=0; Path=/
Vary: Cookie
Set-Cookie: session=eyJfZmxhc2hlcyI6W3siIHQiOlsiZGFuZ2VyIiwiSW52YWxpZCBlbWFpbCBvciBwYXNzd29yZCJdfV0sImNzcmZfdG9rZW4iOiI5YTk2MDk1NGFmODFiNWNiNzU1ZjAxMzNmNDBkN2FlZGUxZGIxNDUzIn0.YDp9_g.mS8JKXm7oj3848fhUYMCexc_WdQ; HttpOnly; Path=/
Server: Werkzeug/1.0.1 Python/3.8.8
Date: Sat, 27 Feb 2021 17:14:38 GMT
```

Invalid email or password

**Comment by crmassociates** [comment]
When I got to step 4 above, I ran into some issues as well.

I solved this by using docker desktop and opening the CLI for natlas_server1 ( your naming may be different ).
Once I got the # prompt I did an `export SERVERNAME=localhost:5000`
followed by:   `flask user new --admin`

no idea if this will persist yet as I'm brand new to containerization, but I have no issues logging out or logging back in to the application by doing this.
